### PR TITLE
Chapter 14

### PR DESCRIPTION
**Pull request by bast**
This starts with a refactor of OpenMP examples since I will reuse them in chapter 14.

Planning for 3-4 recipes:
- Visual studio
- Possibly Ninja
- Simple Linux -> Windows cross-compilation
- Linux -> Windows cross-compilation with OpenMP

### CODE REVIEW

File: chapter-03/recipe-05/fortran-example-3.5/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# set minimum cmake version
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+
+# project name and language
+project(recipe-05 LANGUAGES Fortran)
+
+find_package(OpenMP REQUIRED)
+
+# We do this in a pedestrian way because the Fortran support is relatively recent
+if(DEFINED CMAKE_Fortran_COMPILER_ID AND NOT DEFINED OpenMP_Fortran_FLAGS)
```
Comment (robertodr) [inline chapter-03/recipe-05/fortran-example-3.5/CMakeLists.txt]: OpenMP flags for Fortran are part of CMake 3.5, so this whole blurb can be (finally!) removed.

Comment (bast) [reply chapter-03/recipe-05/fortran-example-3.5/CMakeLists.txt]: Cool! I will nuke the blurb.

**Comment by bast** [comment]
I will submit the chapter 3 changes as independent PR so that it does not have to wait for the entire chapter 14 and we minimize divergence.

**Comment by bast** [comment]
Chapter now contains 2 recipes:
- hello world example
- OpenMP example

The first will fill two recipes in the book: we will build it using VS but then also cross-compile it from Linux to Windows.

**Comment by bast** [comment]
I will wait for Travis and then merge. Work remains but is followed up in #321.

**Comment by bast** [comment]
Merging. There is a lot of blood on the CI floor but all looks unrelated to these changes.

**Pull request merged**